setup.cfg

```diff
@@ -12,10 +12,9 @@ include_trailing_comma = true
 
 length_sort = false
 multi_line_output = 3
-not_skip = __init__.py
```

Comment:
Due to https://timothycrosley.github.io/isort/docs/upgrade_guides/5.0.0/ we can just remove this